gh-skeleton

```diff
@@ -220,10 +223,12 @@ configure_branch_protection() {
 
   log_info "Determining organization type for $dest_org."
   is_real_org=true
-  gh api "/orgs/$dest_org" --silent || is_real_org=false
+  gh api "/orgs/$dest_org" --silent 2> /dev/null || is_real_org=false
   if [ $is_real_org = true ]; then
+    log_ok "Organization $dest_org is a true organization."
     configure_org_branch_protection "$dest_repo" "$dest_org" "$branch"
   else
+    log_ok "Organization $dest_org is a personal account."
```

Comment:
Is "personal account" the only other possibility if it's not a true organization?  If so, this is fine.  If not, we should re-word this log message.  I'm mainly thinking of GH Enterprise accounts as a potential third option here.  https://docs.github.com/en/get-started/learning-about-github/types-of-github-accounts

Reply:
I see what you're saying.   But in GitHub parlance an "enterprise" is a group of "organizations".  It is a different beast, and lives in a different part of the API:  https://docs.github.com/en/rest/enterprise-admin/audit-log

You'll never clone a repo to or from an enterprise account, so I think that this log message is accurate and can stay as it is.

Reply:
I suspected that was the case, but thank you for confirming.